### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zlydh**
Update CHANGELOG.md for M29 release.

### CODE REVIEW

File: Firebase/Messaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2015-07-10 -- v3.0.3
+- Clean up unused `FIRMessaging_PROBER` macro.
```
Comment (chliangGoogle) [inline Firebase/Messaging/CHANGELOG.md]: Move this to the last one: Remove unused heart beat time stamp tracking.

File: Firebase/Messaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Updated topic subscription warning.
```
Comment (chliangGoogle) [inline Firebase/Messaging/CHANGELOG.md]: Corrected the deprecation warning when subscribing to or unsubscribing from an invalid topic.

File: Firebase/Messaging/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+- Add a feature that queues up the topic requests when token is not available and syncs the topic when the token is generated.
```
Comment (chliangGoogle) [inline Firebase/Messaging/CHANGELOG.md]: Fixed an issue that client should suspend the topic requests when token is not available and resume the topic operation when the token is generated.

And move this to the top.

File: Firebase/Messaging/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 2015-07-10 -- v3.0.3
+- Fixed an issue that client should suspend the topic requests when token is not available and resume the topic operation when the token is generated.
+- Corrected the deprecation warning when subscribing to or unsubscribing from an invalid topic.
```
Comment (paulb777) [inline Firebase/Messaging/CHANGELOG.md]: Add "(#1397)" to make it clear which issue was fixed.

**Pull request merged**